README.md

```diff
@@ -1,6 +1,11 @@
 # nwoc_algorithms
 This is a library of various algorithms that are used in competitive programming.
 
+#Combination computation
+[combination_by_elurer_method.cpp](Algorithms/Combination/combination_by_elurer_method.cpp) is a algorithm to compute the combinations of large numbers. Since in competitive programming we often deal with large numbers so computing combination becomes difficult so therefore 
```

Comment:
Page not found on Algorithms/Combination/combination_by_elurer_method.cpp please check